Project3/Part_A/main.cpp

```diff
@@ -3,12 +3,16 @@
 #include "Dictionary.h"
 #include "Grid.h"
 
-const int MIN_LENGTH = 5;
+const int MIN_LENGTH = 3;
 
 void findMatches(Dictionary dict, Grid grid);
-void search(Grid grid, int row, int col, std::vector<std::string> found);
+void search(Dictionary dict, Grid grid, int row, int col);
```

Comment:
would a name such as searchLocation be clearer? Not a neccessary change, but if you're making changes anyway...